### DISCUSSION TITLE
Reaktoro package for handling chemical reactions

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [PMEAL/OpenPNM#1990] [Development]
https://reaktoro.org/overview.html

**Comment by volpatto** [comment]
Hi, OpenPNM team!

Firstly, thank you for this great lib! I am quite interested in this subject, but it seems like it has no priority (no blaming intended, don't get me wrong). Do you guys have any perspective on the continuation of this idea? Cheers.

**Comment by jgostick** [comment]
Sorry, there is no progress on this.  This discussion item is mostly just a reminder for us to think about it someday.

**Reply by volpatto** [reply]
> Sorry, there is no progress on this. This discussion item is mostly just a reminder for us to think about it someday.

@jgostick would be convenient if I make a prototype (like a tutorial, ipynbs, etc) to start this in the future? I'm planning to give it a shot this year.

**Reply by ma-sadeghi** [reply]
We’d definitely welcome a notebook. If it looks promising, we might consider adding it to the package.
